### ISSUE TITLE
Change filesystem to NTFS support

### ISSUE THREAD

**Issue by ElXreno** [omnirom/android_bootable_recovery#262]
- [ ] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [*] I am running the latest version of TWRP
- [*] I have read the FAQ (https://twrp.me/FAQ/)
- [*] I have searched for my issue and it does not already exist

**Device codename**: [M3s](https://github.com/ElXreno/twrp_device_meizu_m3s)
**TWRP version**: 3.3.1

I enabled NTFS support in my BoardConfig, build it, and it works. But, in example, if i want to change filesystem on some partition or flash drive to NTFS, i cannot do this because button "NTFS" on Change File System page doesn't exists. 
Can you fix this?

P.S. I'm not sure, but maybe i can change this in code, and send pull request. This code will be ugly, i think.

**Comment by MartinX3** [comment]
Be aware that NTFS is a filesystem with journaling, designed for normal hard disk  drives, not flash drives.
It ages your flash cells faster, that means that your flash drive/UFS/smartphone storage/sdcard dies faster and is slower.

**Comment by ElXreno** [comment]
@MartinX3 Yes, i agree with you. I hate NTFS (and few other filesystems) but maybe it would still be worth adding it to that list? Some people would like to have this opportunity.

**Comment by MartinX3** [comment]
Yes, of course, the freedom of choice.
But sadly without enough warnings people may decide not so good things.
I hope they won't come and blame twrp :S

BTW I am still on the search for a good file system.
Fat32 has good open source support, but is an old file system with not great metadata support.
F2fs may be slower, btfrs may be not good either I did hear, ext4 is not default supported for sdcards in aosp, exfat is proprietary and we need the work from relan on github.
Hmmmm.
Specially for "syncthing".

**Comment by ElXreno** [comment]
@MartinX3 I think twrp cannot be guilty since they do everything at their own peril and risk, not twrp itself :)

About file systems. I'm using ext4. In my opinion, ext4 faster and stable than everyone else file system.  I once used btrfs, and n this file system, everything is fine, except for one thing: unstable. And i cannot recover it. Uh.
Maybe better use ext2\ext3 on sdcards, but... I want to be sure that everything will be fine, and everything can be restored.

**Comment by MartinX3** [comment]
For sure, but sadly sometimes people blame for becoming the ability to decide. :P

Thanks for the info :)
Hmm, wasn't ext4 somehow flash cell aware and supports them somehow with optimizations?

**Comment by ElXreno** [comment]
@MartinX3 I use cyanogenmod, and laptop with Fedora, it's not a problem :)

**Comment by MartinX3** [comment]
@ElXreno I hope I can put in AOSP and OmniROM in the future if I gather some free time to do it and read into it how to do it, i hate the divergence of useful features. :)
